pkg/utils/net-attach-def.go

```diff
@@ -122,7 +122,7 @@ func GetNetworkStatus(pod *corev1.Pod) ([]v1.NetworkStatus, error) {
 }
 
 // CreateNetworkStatus create NetworkStatus from CNI result
-func CreateNetworkStatus(r cnitypes.Result, networkName string, defaultNetwork bool) (*v1.NetworkStatus, error) {
+func CreateNetworkStatus(r cnitypes.Result, networkName string, defaultNetwork bool, dev *v1.DeviceInfo) (*v1.NetworkStatus, error) {
```

Comment:
This is a totally valid change, and I'm 100% OK with it. 

But it makes me realize that we don't yet have any tags for this repo. So, with a change for a publicly available method... I think we might want to tag a version before we merge this in. I'm adding some notes to the NPWG agenda to see where we should start with tagging a version.

Reply:
Added release v1.1.0 @ https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/releases/tag/v1.1.0